MekHQ/src/mekhq/campaign/mission/Loot.java

```diff
@@ -107,11 +107,15 @@ public void clearParts() {
     }
 
     public String getShortDescription() {
-        String desc = getName() + " - ";
+        String desc = getName() + "  ";
         if (cash.isPositive()) {
             desc += cash.toAmountAndSymbolString();
         }
 
+        if (cash.isNegative()) {
```

Comment:
At this point, you might as well just remove the if blocks and always append cash.toAmountAndSymbolString() to desc (unless it's zero, I guess).